docs/diginstroom/sip/2.1/usecases/standard-film-1-image-reel.md

```diff
@@ -0,0 +1,1173 @@
+  <dcterms:license>BEZOEKERTOOL-CONTENT</dcterms:license>
+  <dcterms:license>BEZOEKERTOOL-METADATA-ALL</dcterms:license>
+
+</metadata>
+```
+
+### data/metadata/preservation/premis.xml
+
+The `premis.xml` of the package level describes the IE and the relationships with its representations.
+It also contains relevant events that detail the digitization of the film, and its lifecycle up until the ingest of its digitized counterpart into meemoo's digital archives.
+
+Note that the identifier in the `<premis:objectIdentifier>` element is shared with the `<dcterms:identifier/>` (in the `descriptive/dc+schema.xml` file) in order to link the PREMIS IE object to its description.
+
+Finally, attention should be paid to the 'carrier representation' that was briefly introduced in the previous section.
+This representation is a construct based on a `premis:representation` object that is entirely dedicated to the analog representation of the film (i.e. the image reel that was digitized).
+Using a number of `<premis:significantProperties>` elements, this representation allows to describe the analog carrier in great detail and to use it as the source of a number of events (e.g. inspection, registration, digitization...).
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<premis:premis version="3.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns:premis="http://www.loc.gov/premis/v3" xmlns:haObj="https://data.hetarchief.be/ns/object/"
+  xsi:schemaLocation="http://www.loc.gov/premis/v3 https://www.loc.gov/standards/premis/premis.xsd">
+
+  <!-- IE for the film as a whole -->
+  <premis:object xsi:type="premis:intellectualEntity">
+
+    <premis:objectIdentifier>
+      <premis:objectIdentifierType>UUID</premis:objectIdentifierType>
+      <premis:objectIdentifierValue>uuid-f9ef158c-f03c-4840-836e-8ffb8e8ebe04</premis:objectIdentifierValue>
+    </premis:objectIdentifier>
+
+    <premis:objectIdentifier>
+      <premis:objectIdentifierType>MEEMOO-LOCAL-ID</premis:objectIdentifierType>
+      <premis:objectIdentifierValue>2891#422</premis:objectIdentifierValue>
+    </premis:objectIdentifier>
+
+    <premis:objectIdentifier>
+      <premis:objectIdentifierType>MEEMOO-PID</premis:objectIdentifierType>
+      <premis:objectIdentifierValue>kiodik2z9x</premis:objectIdentifierValue>
+    </premis:objectIdentifier>
+
+    <!-- relationship with the (abstract) carrier representation -->
+    <premis:relationship>
+      <premis:relationshipType authority="relationshipType"
+        authorityURI="http://id.loc.gov/vocabulary/preservation/relationshipType"
+        valueURI="http://id.loc.gov/vocabulary/preservation/relationshipType/str">structural</premis:relationshipType>
+      <premis:relationshipSubType authority="relationshipSubType"
+        authorityURI="http://id.loc.gov/vocabulary/preservation/relationshipSubType"
+        valueURI="http://id.loc.gov/vocabulary/preservation/relationshipSubType/isr">is represented
```

Comment:
in the meantime, we introduced https://data.hetarchief.be/ns/object/hasCarrierCopy

Reply:
Updated UC, sample and specification accordingly.